docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -46,108 +46,93 @@ FSNetCore/
 
 ### Writing a Class library
 
-1. Create a `Library` folder under `FSNetCore/src`.
-2. In the command line, execute `dotnet new -l F# -t lib` in `FSNetCore/src/Library`.
-3. Replace the contents of `Library.fs` with the following:
-
-    ```fsharp
-    module Library
-
-    open Newtonsoft.Json
-
-    let getJsonNetJson value = 
-        sprintf "I used to be %s but now I'm %s thanks to JSON.NET!" value  (JsonConvert.SerializeObject(value))
-    ```
-
-5. Replace the contents of `project.json` with the following:
-
-    ```json
-    {
-      "version": "1.0.0-*",
-      "buildOptions": {
-        "debugType":"portable",
-        "compilerName": "fsc",
-        "compile": {
-          "includeFiles": [
-            "Library.fs"
-          ]
-        }
-      },
-      "tools": {
-        "dotnet-compile-fsc":"1.0.0-preview2-*"
-      },
-      "frameworks": {
-        "netstandard1.6": {
-          "dependencies": {
-            "NETStandard.Library":"1.6.0",
-            "Microsoft.FSharp.Core.netcore":"1.0.0-alpha-160629",
-            "Newtonsoft.Json":"9.0.1"    
-          }
-        }
-      }
-    }
-    ```
-
-6. Run `dotnet restore` and `dotnet build`.  These should succeed.
+Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
+
+```bash
```

Comment:
bash is not a supported language identifier for docs so the colorization is funky on this command.